mcp/gae_locals.tf

```diff
@@ -45,38 +45,24 @@ locals {
   }
   //noinspection HILUnresolvedReference
   as_paths = {
-    for as, spec in local.as_all_specs:
+    for as, spec in local.as_all_specs :
       as => {
-        build_dir: format("../%s/%s", lookup(spec, "root_dir", as), lookup(spec, "build_dir", "build"))
-        manifest: format("../%s/%s/mmcf-manifest.json", lookup(spec, "root_dir", as), lookup(spec, "build_dir", "build"))
-      }
+        build_dir : format("%s/%s", lookup(spec, "root_dir", as), lookup(spec, "build_dir", "build"))
```

Comment:
Since `local.as_paths.build_dir` wasn't used anywhere, changed the format so it could be used as a variable when making `local.src_files`